server/http4s-server/src/test/scala/tapir/server/http4s/Http4sServerTests.scala

```diff
@@ -0,0 +1,47 @@
+package tapir.server.http4s
+import cats.data.Kleisli
+import cats.effect.concurrent.Ref
+import cats.effect._
+import fs2.concurrent.SignallingRef
+import org.http4s.server.blaze.BlazeServerBuilder
+import org.http4s.syntax.kleisli._
+import org.http4s.{Request, Response}
+import tapir.Endpoint
+import tapir.server.tests.ServerTests
+import tapir.typelevel.ParamsAsArgs
+import cats.implicits._
+import scala.concurrent.ExecutionContext
+
+class Http4sServerTests extends ServerTests[IO] {
+
+  implicit val ec: ExecutionContext = scala.concurrent.ExecutionContext.Implicits.global // Is this the one we want?
```

Comment:
yes, it's usually safe to use, esp in tests :)

Reply:
Maybe we should extract this to a trait? We need it  both here and in client tests.

Reply:
I don't think there's any module that depends on cats and is shared between client and server tests?

Reply:
Http4s is used in client tests?

Reply:
yes, to provide a server to which the clients can talk to.

Reply:
Just to be sure — you do agree with bumping the version of http4s to 0.20.0-m4 everywhere?

Reply:
yes sure :)